helidon-cli/impl/src/test/java/io/helidon/build/cli/impl/TestUtils.java

```diff
@@ -75,18 +76,31 @@ static String resourceAsString(String name) {
     static ExecResult exec(String... args) throws IOException, InterruptedException {
         List<String> cmdArgs = new ArrayList<>(List.of(javaPath(), "-cp", "\"" + System.getProperty("java.class.path") + "\""));
         String version = System.getProperty(HELIDON_VERSION_PROPERTY);
+        return execWithDirAndInput(null, null, args);
+    }
+
+    static ExecResult execWithDirAndInput(File wd, File input, String... args) throws IOException, InterruptedException {
+        List<String> cmdArgs = new ArrayList<>();
+        cmdArgs.addAll(List.of(javaPath(), "-cp", "\"" + System.getProperty("java.class.path") + "\""));
+        String version = System.getProperty(HELIDON_VERSION_PROPERTY);
         if (version != null) {
             cmdArgs.add("-D" + HELIDON_VERSION_PROPERTY + "=" + version);
         }
         cmdArgs.add(Main.class.getName());
         cmdArgs.addAll(Arrays.asList(args));
         ProcessBuilder pb = new ProcessBuilder(cmdArgs);
+        if (wd != null) {
+            pb.directory(wd);
+        }
+        if (input != null) {
+            pb.redirectInput(input);
+        }
         Process p = pb.redirectErrorStream(true).start();
         String output = new String(p.getInputStream().readAllBytes(), StandardCharsets.UTF_8);
         if (!p.waitFor(10, TimeUnit.SECONDS)) {
             throw new IllegalStateException("timeout waiting for process");
         }
-        return new ExecResult(p.exitValue(), stripAnsi(output));
```

Comment:
Did you run into a case where an assertion required the ansi escapes in the output?

Reply:
I'm guessing you removed it because it should be the case that ansi escapes aren't injected in this configuration. I added this as a guard against future changes that break that assumption.

Reply:
This looks like a merge issue on my part, not intentional. Will fix.